pkgs/development/libraries/discord-gamesdk/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchzip
+, pkg-config
```

Comment:
And a redundant argument, it just slipped below my eye.

Reply:
Fixed in the most recent push.